### ISSUE TITLE
Privacy Enhancements

### ISSUE THREAD

**Issue by rebel58** [system76/laptop-suggestions#103]
### Why/User Benefit/User Problem
Privacy is important

### Description of the feature
Add camera lens shade and include a microphone mechanical off switch

**Comment by 22bsti** [comment]
Agree.  Camera shade like the Thinkpads and newer HPs and hardware mic switch.
